### ISSUE TITLE
SplitFragments: Fragment file without Index

### ISSUE THREAD

**Issue by YU-LIN96** [stuart-lab/signac#1586]
I am trying to use the SplitFrament function to generate a cell-type-based file. And the content looks quite like the atac_fragments.tsv.gz format.

SplitFragments(hip.obj, assay = "ATAC",group.by = "celltype", outdir = "./")

The output files are .bed files:
chr1    3015140 3015497 TTCCCACAGCTGCACA-1      1
chr1    3025451 3025834 AGTGAACCATGACTAT-1      2
chr1    3034384 3034623 TAAGCCTAGTTTGAGC-1      1
chr1    3041824 3042165 ACGAGTAAGCTTACTT-1      1
chr1    3050989 3051204 GGTAACTTCGTGCTTC-1      1
chr1    3060818 3060940 ATTAGCGGTGATGATT-1      1
chr1    3090964 3091017 AGCAGGTAGCAATAGG-1      1
chr1    3094051 3094088 TAATGGACATGCTTAG-1      1
chr1    3094937 3094985 TAATGGACATGCTTAG-1      2
chr1    3096773 3096940 ATAAGCCTCAGGTCCA-1      4

However, these .bed has no index. I then generate the index myself
```
#! /bin/bash

sort -k1,1 -k2,2n unsorted.bed > sorted.bed 
# if unsorted, [E::hts_idx_push] Chromosome blocks not continuous
# tbx_index_build failed: Ast.bed.gz

bgzip -c sorted.bed > sorted.bed.gz
tabix -p bed sorted.bed.gz
```
I then tried to input the fragment file again.

```r
temp <- CreateFragmentObject("./new_Fig_HIP/BW/Ast.sorted.bed.gz")
```

A Fragment object for 0 cells.
This is quite confusing.

I am trying to

**Comment by timoast** [comment]
You are correct that the `SplitFragments` function does not generate the tabix index. This is the expected result, rather than a bug. The main use for this function in Signac is for peak calling, which does not require the index file.

It says `A Fragment object for 0 cells.` because you did not specify the `cells` parameter. If you specify a list of cells, then it will check that these cells are present and list them in the object information.
